internal/pkg/deploy/cloudformation/stack/testdata/workloads/svc-staging.stack.yml

```diff
@@ -427,7 +429,7 @@ Resources: # If a bucket URL is specified, that means the template exists.
     Type: Custom::RulePriorityFunction
     Properties:
       ServiceToken: !GetAtt RulePriorityFunction.Arn
-      RulePath: !Ref RulePath
```

Comment:
Can we not use `!Select [ 0, !Ref RulePath ]` to replace `RulePath` entirely?

Reply:
We can not because `RulePath` is not a list at the moment. And if we make it a list then there are multiple places where we ill need to make change which is anyway going away with the upcoming multiple port PR.